SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/SpatialViewUtils.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+#include "SpatialView/ViewDelta.h"
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+Worker_EntityId TestEntityId = 1;
+Worker_ComponentId TestComponentId = 1;
+double TestComponentValue = 20;
+double OtherTestComponentValue = 30;
+double TestEventValue = 25;
+FString DisconnectReason = TEXT("Test disconnection reason");
+
+void SetFromOpList(ViewDelta& Delta, EntityView& View, EntityComponentOpListBuilder& OpListBuilder)
+{
+	OpList Ops = MoveTemp(OpListBuilder).CreateOpList();
+	TArray<OpList> OpLists;
+	OpLists.Push(MoveTemp(Ops));
+	Delta.SetFromOpList(MoveTemp(OpLists), View);
+}
+
+inline void AddEntityToView(EntityView& View, const Worker_EntityId EntityId)
+{
+	View.Add(EntityId, EntityViewElement());
+}
+
+inline void AddComponentToView(EntityView& View, const Worker_EntityId EntityId, const Worker_ComponentId ComponentId, const double Value)
```

Comment:
Minor point - I'd rather if this took a generic component data rather than a value for the test one as that would be more consistent with the other code in the tests. + it would make this more easily usable elsewhere